content/source/docs/enterprise/system-overview/security-model.html.md

```diff
@@ -0,0 +1,100 @@
+
+## Additional Personas for Terraform Enterprise
+
+In addition to the personas listed in the Terraform Cloud Security Model, Terraform enterprise requires two additional personas for managing and administering the application.
+
+
+### Infrastructure Admin
+
+Outside of the application, administrators of the Terraform Enterprise deployment are responsible for managing the underlying infrastructure, upgrading the application, and configuring Terraform Enterprise either via the [Replicated admin console](../install/config.html#system-configuration) or by editing the [application settings file](../install/automating-the-installer.html).
+
+Due to the extensive capabilities granted to this role, the infrastructure admin role be restricted to a small number of users within your organization.
+
+### Site Admin
+
+[Site admins](../admin/admin-access.html) are responsible for application-level configuration of Terraform Enterprise. Via the admin interface they will be able to manage all users, workspaces and organizations and will have access to all data stored within Terraform Enterprise. Site admins are also responsible for configuring SAML and will be the only users still able to access Terraform Enterprise via username/password once SAML is configured. 
+
+Due to the extensive capabilities granted to this role, the site admin role should be restricted to a small number of users within your organization.
+
+
+## Differences Between Terraform Enterprise and Terraform Cloud Security Models
+
+For the most part, Terraform Enterprise’s threat model can be considered a superset of the [Terraform Cloud security model](../../cloud/architectural-details/security-model.html) and (with the exception of the points listed below) all of the content on that page applies to Terraform Enterprise. Terraform Enterprise’s threat model differs from that of Terraform Clouds in these respects:
+
+### TFE Requires You to Manage and Secure the Underlying Network and Infrastructure
+
+TFE infrastructure admins are required to manage all aspects of the underlying infrastructure including initial provisioning, secure configuration, access control, configuring network ACLs, and OS-level software updates. TFE can not ensure the security of your data if the underlying infrastructure is compromised.
+
+### You are Responsible for Updating Your TFE Deployment
+
+Security fixes are released along with application features and bug fixes via TFE’s standard monthly release cycle. TFE infrastructure admins are responsible for applying updates.
```

Comment:
I'm also wondering - is there a release notes page/section that we could link to here?

```suggestion
We release security fixes, application features, and bug fixes for Terraform Enterprise each month. Infrastructure admins are responsible for applying updates.
```